addon/components/rdf-input-fields/numerical-input.js

```diff
@@ -11,8 +11,13 @@ export default class RdfInputFieldsNumericalInputComponent extends SimpleInputFi
   updateValue(e) {
     if (e && typeof e.preventDefault === 'function') e.preventDefault();
     this.value = e.target.value;
-    const number = literal(Number(this.value), this.datatype);
-    super.updateValue(number);
+    if (this.value == '' && this.args.field.options.allowClear) {
```

Comment:
Does this actually need to be an option, this sounds more like a bugfix to me, actually? Is there ever a use-case where you don't want this to happen?

Reply:
We wanted to do this because of backwards compatibility, since this has been in the codebase for so long, it could be specific applications expect this behaviour.

Reply:
I follow @Windvis here. I wouldn't add this extra option. It was basically a bug at the start. Maybe put it in a release note?